media/js/src/simulationOne.jsx

```diff
@@ -13,7 +13,18 @@ export const SimulationOne = () => {
     const [userBetaOneHat, setUserBetaOneHat] = useState(0);
     const [userSE, setUserSE] = useState(0);
     const [userBetaOne, setUserBetaOne] = useState(0);
-    const [userBetaOneTwo, setUserBetaOneTwo] = useState(0);
+    const [valueMatch, setValueMatch] = useState(false);
+
+    useEffect(() => {
+        // Check if all values are not null before performing comparison
+        if (intercept !== null) {
+            setValueMatch(
+                parseFloat(intercept.toFixed(3)) === parseFloat(userBetaOneHat)
```

Comment:
Interesting.. I've never really had to compare floats before. This seems to be a good solution. And a more lenient solution could be making sure the difference between the two numbers is `< 0.001` or some arbitrary amount like that.